infra/profile.py

```diff
@@ -129,17 +123,19 @@ def aggregate(compile_times, bench_times, output_directory):
     print("Usage: profile.py <bril_directory> <output_directory>")
     exit(1)
 
-  profile_path, output_path = os.sys.argv[1:]
+  # Create tmp directory for intermediate files
+  os.mkdir(TMP_DIR)
```

Comment:
If this script is killed and the tmp dir is still there, does this fail? Do we want to delete it in that case?